### ISSUE TITLE
A question rather then an issue

### ISSUE THREAD

**Issue by tomaszhanc** [proophsoftware/fee-office#3]
Hi, it's a question, not an issue. You are using path repositories for your modules: https://github.com/proophsoftware/fee-office/blob/master/composer.json#L38. It's great because during the development all modules are symlinked in a `vendor` directory. But what about building an artifact during deployment by `composer archive`? All symlinked modules would be gone. I mean they won't be included in a tar file. How did you handle that?

**Comment by codeliner** [comment]
Hi, see composer docs: https://getcomposer.org/doc/05-repositories.md#path

> Instead of default fallback strategy you can force to use symlink with "symlink": true or mirroring with "symlink": false option. Forcing mirroring can be useful when deploying or generating package from a monolithic repository.

**Comment by tomaszhanc** [comment]
@codeliner yes, but then I can't use symlink during development which is really convenient. I think it's not possible, but what I would like to achieve it to use `composer install` during development and then install path repositories using a symlink and then use `composer install --some-option` to install path repositories as a hard copy. Thanks for answer!

**Comment by codeliner** [comment]
You could add a build pipeline step to switch the corresponding flag in the composer.json. `composer install --some-option` would be better, but a pipeline step should do the trick as well.
